pkg/server/server.go

```diff
@@ -85,7 +85,7 @@ func New(log logr.Logger, restConfig *rest.Config, cm certmanager.Signer, tls tl
 
 	meshcnf := mesh.DefaultMeshConfig()
 	meshcnf.TrustDomain = tls.TrustDomain()
-	auther := kubeauth.NewKubeJWTAuthenticator(mesh.NewFixedWatcher(&meshcnf), kubeClient, cluster.ID(opts.ClusterID), nil, jwt.PolicyThirdParty)
+	auther := kubeauth.NewKubeJWTAuthenticator(mesh.NewFixedWatcher(meshcnf), kubeClient, cluster.ID(opts.ClusterID), nil, jwt.PolicyThirdParty)
```

Comment:
Required due to the bump in istio version. Superficial pointer pointer to pointer.